Tests/SonarScanner.MSBuild.Shim.Test/PropertiesWriterTest.cs

```diff
@@ -599,34 +529,23 @@ public void PropertiesWriter_WorkdirPerModuleExplicitlySet()
         var props = new JavaProperties();
         props.Load(GenerateStreamFromString(s));
         var key = projectKey + "." + SonarProperties.WorkingDirectory;
-#pragma warning disable DictionaryShouldContainKey // Simplify Assertion
         props.ContainsKey(key).Should().BeTrue();
-#pragma warning restore DictionaryShouldContainKey // Simplify Assertion
-    }
 
-    public static Stream GenerateStreamFromString(string s)
-    {
-        var stream = new MemoryStream();
-        var writer = new StreamWriter(stream);
-        writer.Write(s);
-        writer.Flush();
-        stream.Position = 0;
-        return stream;
+        static Stream GenerateStreamFromString(string s)
+        {
+            var stream = new MemoryStream();
+            var writer = new StreamWriter(stream);
+            writer.Write(s);
+            writer.Flush();
+            stream.Position = 0;
```

Comment:
StreamWriter is disposable and flushes on Dispose.
```suggestion
            using (var writer = new StreamWriter(stream))
            {
                writer.Write(s);
            }
            stream.Position = 0;
```

Reply:
This breaks the test ;_;